ui/src/test/java/com/hyperwallet/android/ui/transfermethod/TransferMethodUtilsTest.java

```diff
@@ -134,4 +140,48 @@ public void testGetStringFontIcon_returnsDefaultValue() {
         verify(mResources, times(2)).getIdentifier(anyString(), anyString(), anyString());
         verify(mContext, times(1)).getString(anyInt());
     }
+
+    @Test
+    public void testGetSecondLine_returnsPayPalSecondLine() throws JSONException {
```

Comment:
Please keep the test name consistent with the method name, given the method name getTransferMethodDetail,
the test names should follow the pattern: 
testGetTransferMethodDetail_returnsPayPalDetail
testGetTransferMethodDetail_returnsCardDetail
testGetTransferMethodDetail_returnsBankAccountDetail
testGetTransferMethodDetail_returnsPaperCheckDetail

Reply:
renamed